pkg/sessions/redis/redis_store.go

```diff
@@ -127,7 +128,10 @@ func buildStandaloneClient(opts options.RedisStoreOptions) (Client, error) {
 	}
 
 	if opts.InsecureSkipTLSVerify {
-		opt.TLSConfig.InsecureSkipVerify = true
+		/* #nosec */
+		opt.TLSConfig = &tls.Config{
+			InsecureSkipVerify: true,
+		}
```

Comment:
I think this just needs a not nil check, eg
```
if opt.TLSConfig == nil {
  opt.TLSConfig = &tls.Config{}
}
opt.TLSConfig.InsecureSkipVerify = true
```
If you did this below as well it would make sure you don't clobber any other options

Reply:
You're right. Thanks for the comment!